### PR TITLE
[#1641] add parent option to createTag

### PR DESCRIPTION
**Pull request by npeltier**
- add options parameter to createTag signature, with only one parent option for now, as discussed in https://github.com/orgs/adobecom/discussions/1641,
- add unit test for create Tag

Resolves: https://github.com/orgs/adobecom/discussions/1641

Before: https://main--milo--adobecom.hlx.page/?martech=off

After: https://discussion_1641_utils--milo--npeltier.hlx.page/?martech=off

### CODE REVIEW

**Review by mokimo — APPROVED** [review]

File: libs/utils/utils.js [hunk 1]
```diff
@@ -257,6 +257,9 @@ export function createTag(tag, attributes, html) {
       el.setAttribute(key, val);
     });
   }
+  if (options.parent) {
+    options.parent.append(el);
+  }
```
Comment (overmyheadandbody) [inline libs/utils/utils.js:262]: I don't remember if we discussed this in the initial PR, but couldn't we go with `options.parent?.append(el);` and save a couple of bits?

Comment (npeltier) [reply libs/utils/utils.js:262]: :) we sure can

**Comment by npeltier** [comment]
done @overmyheadandbody

**Review by overmyheadandbody — APPROVED** [review]

File: libs/utils/utils.js (hunk 1 above)
Comment (hparra) [reply libs/utils/utils.js]: I'm on the fence but coverage will still recognize the branch, so that's good.

**Review by hparra — APPROVED** [review]
One unrelated test is failing, but that's the way it is... Please remember to add a test page url(s) so that check passes.

**Pull request merged**